procrastinate/sql/schema.sql

```diff
@@ -283,13 +301,30 @@ BEGIN
 END;
 $$;
 
-CREATE FUNCTION procrastinate_notify_queue()
+CREATE OR REPLACE FUNCTION procrastinate_notify_queue_job_inserted()
```

Comment:
Do we really need `CREATE OR REPLACE` in the schema.sql? It's no harm, but I think just `CREATE` should be enough. (I think it's also not necessary for `procrastinate_finish_job`).

Reply:
I guess not. I may as well replace it there too to avoid ambiguity in the future